frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -157,6 +189,9 @@ const GitSection: React.FC<GitSectionProps> = ({ showSample, builderImages }) =>
     if (validated === ValidatedOptions.warning) {
       return 'URL is valid but cannot be reached. If this is a private repository, enter a source secret in Advanced Git Options';
     }
+    if (validated === ValidatedOptions.warning) {
+      return 'Valid URL but devfile build guidance not available.';
+    }
```

Comment:
Both these if conditions check for `warning`. Looks like the condition for not reachable should be an `error` looking at the logic above in this file

Reply:
Interesting, I'll have to validate the UI code once I move it over. I'm trying to close off the ticket I'm working on but clusters seem to be having some issues today. Good catch though.